lib/virt_autotest/common.pm

```diff
@@ -52,7 +52,7 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             macaddress => '52:54:00:78:73:a3',
             ip => '192.168.122.102',
             distro => 'SLE_15',
-            location => 'http://mirror.suse.cz/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/',
+            location => 'http://dist.nue.suse.com/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/',
```

Comment:
Don't know for sure where dist.nue.suse.com is located, but it sounds very much also like NUE.